src/main/scala/com/gu/config/Configuration.scala

```diff
@@ -29,4 +29,6 @@ object Configuration {
   val payPalConfigProvider = new PayPalConfigProvider(config, stage)
   val salesforceConfigProvider = new SalesforceConfigProvider(config, stage)
   val zuoraConfigProvider = new ZuoraConfigProvider(config, stage)
+
+  val contributionThanksQueueName = config.getString("email.thankYou.queueName")
```

Comment:
Should you add a default value to https://github.com/guardian/support-workers/blob/master/src/main/resources/application.conf ?